docker-compose.yml

```diff
@@ -25,6 +26,7 @@ services:
     build:
       context: .
       dockerfile: .docker/Dockerfile.cli
+    # Name the image to access it via CLI_IMAGE arg.
```

Comment:
redundant comment - anyone who knows docker knows what it is.

Reply:
We don't always have people who know docker. Eg me trying to improve drupal DX and having to submit docker patches to chip away at the problem :)

Reply:
Agree on this as a general point, don't make the assumption that govCMS devs are going to have familiarity with Docker.  We've spoken to a number of agencies that have never used Docker before in any context.